src/main/java/gregtech/api/metatileentity/multiblock/RecipeMapMultiblockController.java

```diff
@@ -161,13 +161,4 @@ public void renderMetaTileEntity(CCRenderState renderState, Matrix4 translation,
         super.renderMetaTileEntity(renderState, translation, pipeline);
         this.getFrontOverlay().render(renderState, translation, pipeline, getFrontFacing(), recipeMapWorkable.isActive());
     }
-
```

Comment:
Please also remove now unused import from this file.

Reply:
Done.